message.py

```diff
@@ -1,4 +1,12 @@
-from enum import Enum
+import enum
+
+
+class MessageType(enum.Enum):
+    OFF = enum.auto()
```

Comment:
Instead of using `enum.auto()`, you can use the protocol's number values, then use these when building the string instead of hardcoding it further down.